doc/docs/master/how-tos/basic-data-operations/sql-ingest.md

```diff
@@ -66,20 +97,40 @@ Where:
 
 | Parameter     | Description | 
 | ------------- |-------------| 
-| **protocol**   | The name of the database protocol. As of today, you have the ability to connect to  `postgres`or `mysql`. Note that `mysql` allows you to connect to mariadb. |
+| **protocol**   | The name of the database protocol. <br> As of today, we support: <br>- `postgres` : connect to Postgresql or compatible (for example Redshift).<br>- `mysql` : connect to MySQL or compatible (for example MariaDB). <br>- `snowflake` : connect to Snowflake. |
 | **username**  | The user used to access the database.|
 | **host**      | The hostname of your database instance.|
 | **port**      | The port number your instance is listening on.|
 | **database**  | The name of the database to connect to. | 
 
+
+
+!!! Attention "Snowflake users, you will need a variant of the url above."
+     Pachyderm supports two connection URL patterns to query Snowflake: 
+
+      - `snowflake://username@<account_identifier>/<db_name>/<schema_name>?warehouse=<warehouse_name>`
+      - `snowflake://username@hostname:port/<db_name>/<schema_name>?account=<account_identifier>&warehouse=<warehouse_name>`
+
+     where:
+
+      - *[`account_identifier`](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html)* takes the following form for most URLs: `organization_name`-`account_name` .
```

Comment:
Snowflake `account_identifier` can also be `<organization_name>-<account_name>.snowflakecomputing.com`. This is important to support because the user might already have an existing Snowflake URL, and they just want to copy paste it into our system.